corehq/warehouse/etl.py

```diff
@@ -54,8 +62,9 @@ def _table_context(cls, start_datetime, end_datetime):
         for dep in cls.dependencies():
             dep_cls = get_cls_by_slug(dep)
             context[dep] = dep_cls._meta.db_table
-        context['start_datetime'] = start_datetime
-        context['end_datetime'] = start_datetime
+        context['start_datetime'] = start_datetime.isoformat()
```

Comment:
this does work, however i can see where it'd be confusing since the text doesn't look exactly the same as the text that is stored in postgres. my guess is pg auto converts it. do you have a better practice?

![screen shot 2017-06-22 at 10 41 23 am](https://user-images.githubusercontent.com/918514/27440090-a804ffae-5737-11e7-8872-0e85ef767bcf.png)
![screen shot 2017-06-22 at 10 41 30 am](https://user-images.githubusercontent.com/918514/27440089-a7fcb22c-5737-11e7-8607-cc143ce6cd65.png)